pycds/__init__.py

```diff
@@ -158,8 +173,10 @@ class Obs(Base):
     history = relationship("History", backref=backref('obs_raw', order_by=id))
     variable = relationship(
         "Variable", backref=backref('obs_raw', order_by=id))
-    flags = relationship(
-        "NativeFlag", secondary=association_table, backref="flagged_obs")
+    flags = relationship("NativeFlag", secondary=obs_raw_native_flags_t, backref="flagged_obs")
+    # better named alias for 'flags'; don't repeat backref
+    native_flags = relationship("NativeFlag", secondary=obs_raw_native_flags_t)
```

Comment:
These declarations in `Obs` do establish a `backref` in both `NativeFlag` and `PCICFlag`. In each the relation has the same name, `flagged_obs`.

My reasoning for omitting the `backref` in the declaration of `native_flags` was this: A backref establishes an attribute on the other object (`NativeFlag`) that links to the object being declared (`Obs`). Adding a backref to the alias relation would create a second, identical relation **in `NativeFlag`** to `Obs`. That duplicate relation in `NativeFlag` is pointless (there is no naming issue, as there is in `Obs`), so why have it? (Also, what would it be named?)